app/forms/teacher_interface/sanction_confirmation_form.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module TeacherInterface
+  class SanctionConfirmationForm < BaseForm
+    attribute :application_form
+    attribute :confirmed_no_sanctions
+
+    validate :no_sanctions_confirmed
```

Comment:
```suggestion
    validate :confirmed_no_sanctions, presence: true
```

You could have this and then the locale key would be `blank` and avoids needing to have its own method.